source/Jobbr.Dashboard/DashboardBackend.cs

```diff
@@ -63,12 +62,10 @@ public void Start()
                 })
                 .Configure(app =>
                 {
-                    var manifestEmbeddedProvider = new ManifestEmbeddedFileProvider(typeof(DashboardBackend).Assembly);

```

Comment:
If we don't use the embedded zip file, please also remove the embedding from the project file:

```
  <ItemGroup>
    <EmbeddedResource Include="dashboard-app.zip">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </EmbeddedResource>
  </ItemGroup>
```